package.json

```diff
@@ -1,26 +1,19 @@
 {
-  "dependencies": {
-    "array-zip": "^1.0.0",
-    "atom-screenshot": "^0.4.3",
```

Comment:
Like Webshot, it's downloaded only the first time a task needs to take a shot. Avoids long `npm i` if you just need to update the changelog or whatever.